src/pages/cart/CartSummary.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+
+const CartSummary = ({ totals }) => {
+  return (
+    <div className="w-full self-start rounded-lg border bg-white p-8 md:w-1/4">
+      <div className="flex items-center justify-between pb-2">
```

Comment:
Following DRY principle, I would extract `tax`, `shipping fee`, and `total` to a separate compont (eg `CartSummaryItem` or `CartSummaryRow`) because of its repetitive structure.